src/Components/Facility/ConsultationDetails.tsx

```diff
@@ -98,6 +88,35 @@ export const ConsultationDetails = (props: any) => {
   const [hl7SocketUrl, setHL7SocketUrl] = useState<string>();
   const [ventilatorSocketUrl, setVentilatorSocketUrl] = useState<string>();
```

Comment:
We should also be moving the states down to whatever is necessary. socket url's is not used by all tabs and is used only in the "updates" tab. Move the state management to the updates tab.

Reply:
logic moved to ConsultationUpdatesTab